orquesta/events.py

```diff
@@ -148,6 +153,7 @@
     TASK_PAUSED_WORKFLOW_ACTIVE,
     TASK_PAUSED_WORKFLOW_DORMANT,
     TASK_RESUMING,
+    TASK_RERUNNING,
```

Comment:
Remove if these events are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.